easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-6.5a1-CrayPGI-19.10-cuda-10.1.eb

```diff
@@ -0,0 +1,53 @@
+# created by Luca Marsella (CSCS)
+easyblock = "ConfigureMake"
+
+name = 'QuantumESPRESSO'
+version = '6.5a1'
+local_cudaversion = '10.1'
+versionsuffix = '-cuda-%s' % local_cudaversion
+
+homepage = 'http://www.quantum-espresso.org/'
+description = """Quantum ESPRESSO is an integrated suite of computer codes
+ for electronic-structure calculations and materials modeling at the nanoscale.
+ It is based on density-functional theory, plane waves, and pseudopotentials
+  (both norm-conserving and ultrasoft)."""
+
+toolchain = {'name': 'CrayPGI', 'version': '19.10'}
+toolchainopts = {'usempi': True, 'openmp': True, 'verbose': False}
+
+source_urls = ['https://gitlab.com/QEF/q-e-gpu/-/archive/qe-gpu-%s/' % version]
+sources = ['q-e-gpu-qe-gpu-%s.tar.bz2' % version]
+
+builddependencies = [
+    ('cudatoolkit/10.1.105_3.27-7.0.1.1_4.1__ga311ce7', EXTERNAL_MODULE),
+]
+
+dependencies = [
+    ('intel/19.0.1.144', EXTERNAL_MODULE)
+]
+
+# Scalapack disabled: the GPU eigensolver can outperform the parallel CPU one 
+configopts = ' CC=cc F77=pgf90 MPIF90=ftn '
+configopts += ' --enable-openmp --enable-parallel --with-scalapack=no ' 
+configopts += ' --with-cuda=$CUDATOOLKIT_HOME --with-cuda-cc=60 '
+configopts += ' --with-cuda-runtime=10.1 '
+
+# experimental CUDA-Aware MPI: add -D__GPU_MPI in make.inc
+prebuildopts = """
```

Comment:
maybe need a patch file instead here but not mandatory

Reply:
Thanks for the comment @jgphpc: the problem with the patch is that EasyBuild applies it before building. On the other hand, I need to adapt the file `make.inc` after it is generated in the `configure` step, since it does not exist when the build tarball is unpacked.